Makefile

```diff
@@ -174,7 +174,7 @@ build_node_js_api: install_rs_build_toolchain
 .PHONY: test_core_crypto # Run the tests of the core_crypto module including experimental ones
 test_core_crypto: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --release \
-		--features=$(TARGET_ARCH_FEATURE),experimental -p tfhe -- core_crypto::
+		--features=$(TARGET_ARCH_FEATURE),experimental,$(AVX512_FEATURE) -p tfhe -- core_crypto::
```

Comment:
I think the build_core target update is missing ?

Reply:
also here I would like to have both tests, once without avx512 and once with it as it changes the code being executed

Reply:
depending on whether AVX512 support was ON or not

Reply:
I don't quite get it about the both tests thing. You're not happy by having two workflow jobs for testing ?

Reply:
No I mean that we want to run the test once in this target without avx512 so the existing command is ok and then we run conditionally a second command with avx512 support on and nightly toolchain

As both code paths need to be correct